src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -79,7 +79,7 @@ namespace Messaging {
  *
  */
 #ifndef CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK
-#define CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK (1)
+#define CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK (200)
```

Comment:
This value is in ticks, not milliseconds.  A tick is 2^{CHIP_CONFIG_RMP_TIMER_DEFAULT_PERIOD_SHIFT} milliseconds, so what "1" means depends on what that config setting is.

I think it's set to 6 for all our platforms, so a tick is 64 ms.

Reply:
I think we'll want to to transition to using platform defined `ms_to_ticks` portability method over time.

Reply:
Why is this value in units of "ticks" anyways? Seems to me that it should be in milliseconds and then converted to ticks as Martin said, however there is no `ms_to_ticks` function currently

Reply:
@bzbarsky-apple How do you know that's the definition of a tick?

Reply:
By code inspection, sadly, in ReliableMessageManager.cpp, of where `CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK ` is used.